helm/cluster-openstack/templates/_kubeadm_config_template.tpl

```diff
@@ -15,4 +15,32 @@ spec:
             cloud-provider: external
             node-labels: giantswarm.io/node-pool={{ .name }}
           name: '{{ `{{ local_hostname }}` }}'
+      files:
```

Comment:
Is this the same for the KubeadmControlPlaneTemplate and KubeadmConfigTemplate? If so what do you think about moving this definition into `_helpers.tpl` as a template to keep them in sync and reduce noise in these files?

Reply:
I thought about it, then I wasn't sure if we won't add some files that are different between CP and workers. We can call template on definition but that adds up to the complexity. I can do that if you think that's better.

Reply:
I think it would be nicer as a template definition. If we have different files between the two in the future then we can figure out a way to handle it. My eyes tend to glaze over when I hit long sections of config like this. We could at least have one template definition per file.

Reply:
Did it, if you have a suggestion for definition please do so